src/runtime/components/forms/FormGroup.vue

```diff
@@ -1,19 +1,37 @@
 <template>
   <div :class="ui.wrapper" v-bind="attrs">
-    <div v-if="label" :class="[ui.label.wrapper, size]">
-      <label :for="inputId" :class="[ui.label.base, required ? ui.label.required : '']">{{ label }}</label>
-      <span v-if="hint" :class="[ui.hint]">{{ hint }}</span>
+    <div v-if="label || $slots.label" :class="[ui.label.wrapper, size]">
+      <label :for="inputId" :class="[ui.label.base, required ? ui.label.required : '']">
+        <slot v-if="$slots.label" name="label" v-bind="{ error, label, name, hint, description, help }" />
+        <template v-else>{{ label }}</template>
+      </label>
+      <span v-if="hint || $slots.hint" :class="[ui.hint]">
+        <slot v-if="$slots.hint" name="hint" v-bind="{ error, label, name, hint, description, help }" />
+        <template v-else>{{ hint }}</template>
+      </span>
     </div>
-    <p v-if="description" :class="[ui.description, size]">
-      {{ description }}
+
+    <p v-if="description || $slots.description" :class="[ui.description, size]">
+      <slot v-if="$slots.description" name="description" v-bind="{ error, label, name, hint, description, help }" />
+      <template v-else>
+        {{ description }}
+      </template>
     </p>
-    <div :class="[label ? ui.container : '']">
+
+    <div :class="[label ? ui.container : '']" @click="$event.preventDefault()">
```

Comment:
Why put back `@click="$event.preventDefault()"` here? @romhml Didn't we remove this?

Reply:
```suggestion
    <div :class="[label ? ui.container : '']">
```

Yep, I guess this is due to a bad merge with dev.